src/main/java/com/celements/metatag/MetaTagHeaderRole.java

```diff
@@ -0,0 +1,13 @@
+package com.celements.metatag;
+
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import org.xwiki.component.annotation.ComponentRole;
+
+@ComponentRole
+public interface MetaTagHeaderRole {
```

Comment:
rename to MetaTagProviderRole and add `public @NotNull List<MetaTag> getBodyMetaTags();` as well since meta tags are allowed in body too for HTML5